Major proofread of trust.rst (#29)
italia/eidas-it-wallet-docs: docs/en/trust.rst

The Infrastructure of Trust
+++++++++++++++++++++++++++

The EUDI Wallet Architecture Reference Framework (`EIDAS-ARF`_) defines the Trust Model as a *"collection of rules that ensure the legitimacy of the components and the entities involved in the EUDI Wallet ecosystem."*.

> In this section, we define how the Trust Model is implemented in an infrastructure of Trust based on

Reviewer: I love your revision, however I would not use "we" here

Author: @peppelinux what would you use? is the following suggestion better?
```suggestion
This section defines how the Trust Model is implemented in an infrastructure of Trust based on
```

Entity Configuration and Entity Statement
`````````````````````````````````````````

> The Entity Configuration is the federation metadata that an Entity publishes about itself, which is verifiable by a trusted third party. The Entity Configuration is signed, and it can be verified with one of the public keys contained within it, as well as within the Entity Statement issued by the Trust Anchor or its Intermediate. This is defined in a parameter called authority_hints. The Entity Configuration may also contain one or more Trust Marks regarding its issuer.

Reviewer: Here "by" seems that the verifier is the TTP, while we want to say that it is verifiable thanks to a TTP :-)

Author: ```suggestion
The Entity Configuration is the federation metadata that an Entity publishes about itself, which is verifiable thanks to a trusted third party. The Entity Configuration is signed, and it can be verified with one of the public keys contained within it, as well as within the Entity Statement issued by the Trust Anchor or its Intermediate. This is defined in a parameter called authority_hints. The Entity Configuration may also contain one or more Trust Marks regarding its issuer.
```

However, the EUDI Wallet can still obtain a Wallet Attestation Instance issued by its Wallet Provider, which should contain a Trust Chain related to its issuer (Wallet Provider).

Offline Relying Party Metadata
```````````````````````````

Since the Federation Entity Discovery is only applicable in online scenarios, it is possible to include the Trust Chain in the presentation requests that a Relying Party may issue for an EUDI Wallet.

The Relying Party must sign the presentation request, which should include the trust_chain claim in its header parameter, containing the Federation Trust Chain related to itself.

The Wallet Instance that verifies the request issued by the Relying Party can then use the Trust Anchor public keys to validate the entire Trust Chain related to the Relying Party and attest to its reliability.

Furthermore, the Wallet Instance can apply the metadata policy, if available, to filter out any user attributes not attested in the trusted metadata from the Relying Party's request, which is obtained from the Trust Chain.

Privacy Considerations

> ``````````````````````

Author: ```suggestion
----------------------
```
